libs/util/Auxilary.arucas

```diff
@@ -1,27 +1,34 @@
-import AssertionError from util.Errors;
-import ImmutableList from util.Collections;
+import IllegalAccessError, AssertionError from util.Errors;
 
 /**
- * Miscilaneous methods that don't fit into other categories.
+ * Miscilaneous methods that dont fit into other categories.
```

Comment:
Why was this changed lol.
I mean don't is correct.

Reply:
IntelliJ is suffering for that somehow, because /** format is not recognized as comments....